static/themes/SSF/js/role.js

```diff
@@ -53,6 +53,12 @@ $(function (){
                 }
             };
         }
+        var loader = function(){
+            $assignBtn.button('loading');
+        }
+        var reset = function(){
+            $assignBtn.button('reset');
+        }
```

Comment:
Could you change this ? https://github.com/edengci/eden/pull/86/files#diff-531029fa08934309c9bfd555c189ebb9R23
It needs to be like this...
a < b, not a<b
Not your fault ^, but since you're making changes anyways, why not fix that as well ;)